### PR TITLE
SkippedItem handling updates

### PR DESCRIPTION
**Pull request by wrall**
Update documentation about DataConsistencyScore and improve skipped item documentation

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -204,6 +204,9 @@ For more advanced scenarios, you may be able to deploy Azure Active Directory (A
 
 We recommend that the primary address (sometimes referred to as the "User ID") for each user be at the primary domain (such as "will@fabrikaminc.net"). Typically, this means that the primary email address should match between Microsoft 365 or Office 365 and Google Workspace. If any user is provisioned with a different domain for their primary address, then that user should at least have a proxy address at the primary domain. Each user should have their `ExternalEmailAddress` point to the user in their Google Workspace routing domain ("will@gsuite.fabrikaminc.net"). The users should also have a proxy address that will be used for routing to their Microsoft 365 or Office 365 routing domain (such as "will@o365.fabrikaminc.net").
 
+   > [!NOTE]
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:207]:
```suggestion
> [!NOTE]
```

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 2]
```diff
@@ -204,6 +204,9 @@ For more advanced scenarios, you may be able to deploy Azure Active Directory (A
...
+   > It is recommended that MRM and Archive policies are disabled for these users until after the migration for those users has been completed.  When such features remain enabled during migration, there is a chance that some messages will end up being considered "missing" during content verification process.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:208]:
```suggestion
> We recommend that the Default MRM Policy and Archive policies be disabled for these users until after their migration has been completed.  When such features remain enabled during migration, there is a chance that some messages will end up being considered "missing" during the content verification process.
```

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md [hunk 3]
```diff
@@ -26,9 +26,9 @@ There are 4 possible grades that are derived from the DataConsistencyScore.
 
 |Grade|Description|
 |---|---|
-|**Perfect**| No inconsistencies noted during migration. The migration will succeed.|
-|**Good**| At least 1 inconsistency noted, but the data loss was not impactful. For example, if only metadata or folder permissions were lost during migration. The migration will succeed.|
-|**Investigate**|A small amount of significant data loss was detected, caused by some common inconsistency types. You must approve the migration for it to complete.|
+|**Perfect**|No inconsistencies noted during migration. The migration will succeed.|
+|**Good**|At least 1 inconsistency noted, but the data loss was not impactful. For example, if only metadata or folder permissions were lost during migration. The migration will succeed.|
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:30]:
```suggestion
|**Good**|At least 1 inconsistency noted, but the data loss was not impactful. For example, only metadata or folder permissions were lost during migration. The migration will succeed.|
```

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md [hunk 4]
```diff
@@ -61,9 +63,34 @@ For a batch scored as **Poor**, approving the migration allows you to complete a
 
 If the migration fails with a grade of **Poor**, you cannot force the migration to succeed. Please contact Microsoft Support for assistance.
 
+## How to see which items were not migrated
+
+### In the Classic Exchange admin center (Classic EAC)
+
+1. In the Exchange Admin center, click **recipients**, and then click **migration**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:70]:
```suggestion
1. In the Exchange admin center, select **Recipients** > **Migration**.
```

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md [hunk 5]
```diff
@@ -61,9 +63,34 @@ For a batch scored as **Poor**, approving the migration allows you to complete a
...
+
+2. Select the batch you would like to inspect. Click **View details** in the information pane on the right.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:72]:
```suggestion
2. Select the batch you would like to inspect. Choose **View details** in the information pane on the right.
```

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md [hunk 6]
```diff
@@ -61,9 +63,34 @@ For a batch scored as **Poor**, approving the migration allows you to complete a
...
+
+3. Select the user you would like to inspect. Click **Skipped item details** in the information pane on the right.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:74]:
```suggestion
3. Select the user you would like to inspect. Select **Skipped item details** in the information pane on the right.
```

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md (hunk 4 above)
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:70]:
```suggestion
1. In the Exchange admin center, click **Recipients**, and then click **Migration**.
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:70]: Please confirm capitalization of "Recipients" and "Migration".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions only.

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md (hunk 3 above)
Comment (wrall) [reply Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:30]: This statement actually applies better to the first sentence than the second.

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md (hunk 4 above)
Comment (wrall) [reply Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:70]: I copied and pasted this line from another article.  In classic EAC the titles are all lower-case.

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md (hunk 3 above)
Comment (wrall) [reply Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:30]: I'll separate it out into a different column for clarity as to why this is mentioned.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**